jenkins/bootstrap.py

```diff
@@ -98,8 +98,27 @@ def call(cmd, stdin=None, check=True, output=None):
 
 
 def pull_ref(pull):
-    """Return the tag associated with the PR."""
-    return '+refs/pull/%d/merge' % pull
+    """Turn a PR number of list of refs into specific refs to fetch and check out.
+
+    >>> pull_ref('123')
+    (['+refs/pull/123/merge'], ['FETCH_HEAD'])
+    >>> pull_ref('master:abcd,123:effe')
+    (['master', '+refs/pull/123/head:refs/pr/123'], ['abcd', 'effe'])
+    """
+    if isinstance(pull, basestring) and ':' in pull:
+        args = [x.split(':') for x in pull.split(',')]
+        refs = []
+        checkouts = []
+        for name, sha in args:
+            try:
+                num = int(name)
+                refs.append('+refs/pull/%d/head:refs/pr/%d' % (num, num))
+            except ValueError:  # branch spec
```

Comment:
Would it be better to enforce that the first one is a branch spec and all others a PRs?